ddspipe_yaml/src/cpp/YamlReader_types.cpp

```diff
@@ -428,6 +429,80 @@ void YamlReader::fill(
     }
 }
 
+/************************
+* LOGGING CONFIGURATION *
+************************/
+
+template <>
+DDSPIPE_YAML_DllAPI
+void YamlReader::fill<LogFilter>(
+        LogFilter& object,
+        const Yaml& yml,
+        const YamlReaderVersion version /* version */)
+{
+    if (is_tag_present(yml, LOG_FILTER_ERROR_TAG))
```

Comment:
I believe it would be better to extract the verbosity's value without having a specific tag for each option. You can get inspiration from the DDS Router's discovery-trigger.